Sources/StreamChat/Models/Attachments/ChatMessageImageAttachment.swift

```diff
@@ -35,10 +37,38 @@ public struct ImageAttachmentPayload: AttachmentPayload {
             .flatMap { try? JSONEncoder.stream.encode($0) }
             .flatMap { try? JSONDecoder.stream.decode(T.self, from: $0) }
     }
+    
+    /// Creates `ImageAttachmentPayload` instance.
+    ///
+    /// Use this initializer if the attachment is already uploaded and you have the remote URLs. Create the ``AttachmentFile`` type using the local file URL.
+    ///
+    /// - Parameters:
+    ///   - title: A title, usually the name of the image.
+    ///   - imageRemoteURL: A link to the image.
+    ///   - file: The image file size information.
+    ///   - originalWidth: The original width of the image in pixels.
+    ///   - originalHeight: The original height of the image in pixels.
+    ///   - extraData: Custom data associated with the attachment.
+    public init(
+        title: String?,
+        imageRemoteURL: URL,
+        file: AttachmentFile,
```

Comment:
This needs to be optional since a customer can use a custom CDN and they might not have the file info
```suggestion
        file: AttachmentFile?,
```

Reply:
The whole `AttachmentFile` handling is a little bit odd. Other payload types require it (init takes a non-optional value), therefore I followed the same flow.